src/components/CloseButton/CloseButton.module.css

```diff
@@ -0,0 +1,48 @@
+.close-button {
+  width: var(--eds-size-6);
+  height: var(--eds-size-6);
+  border-radius: 50%;
+
+  svg {
+    /* TODO(Icon): use size prop on Icon after that component is done */
+    height: var(--eds-size-4);
+    width: var(--eds-size-4);
+    /* TODO(Icon): use color prop on Icon after that component is done */
+    fill: var(--close-button-icon-color);
+  }
+
+  &:hover {
+    background-color: var(--close-button-icon-hover-color);
+  
+    /* TODO(Icon): fix after Icon component is done */
+    /* stylelint-disable-next-line max-nesting-depth */
+    svg {
+      fill: var(--eds-theme-color-neutral-bold-foreground-inverted);
+    }
+  }
+}
+
+.close-button--variant-brand {
```

Comment:
super super nit: could this be `close-button--brand`? or is `variant-brand` more consistent?